### DISCUSSION TITLE
Remove esm-only warning when serverModuleFormat is esm

### DISCUSSION THREAD

**Discussion by onemen** [remix-run/remix#6766] [Proposals]
From docs for [serverdependenciestobundle](https://remix.run/docs/en/main/file-conventions/remix-config#serverdependenciestobundle):
> This can be useful when consuming ESM only packages in a CJS build
> Here's how you would be able to consume those packages in a CJS app without having to use dynamic imports

When we configure remix with `serverModuleFormat: 'esm'` we don't need to use dynamic imports for ESM only packages

Do you think it is possible to remove the warning in this case?

**Comment by ngbrown** [comment]
Closed via https://github.com/remix-run/remix/pull/6916

**Reply by jscul** [reply]
Is this the warning Remix is getting rid of?

![image](https://github.com/remix-run/remix/assets/22265367/4a7c9846-8756-46ba-87df-dd70e35f0b32)
